openedx/core/djangoapps/content/learning_sequences/tasks.py

```diff
@@ -39,6 +47,11 @@ def _make_section_data(section):
                     usage_key=sequence.location,
                     title=sequence.display_name,
                     inaccessible_after_due=sequence.hide_after_due,
+                    exam=ExamData(
+                        is_practice=sequence.is_practice_exam,
+                        is_proctored=sequence.is_proctored_enabled,
+                        is_timed=sequence.is_timed_exam
```

Comment:
Can we give the `ExamData` fields the exact same name as the fields on the `sequence`. Things are already confusing enough with so many different names.